app/models/event.rb

```diff
@@ -12,11 +12,13 @@
 class Event < ActiveRecord::Base
   translates :title, :introduction, :conclusion
   belongs_to :location
-  # suggestion from Nicholas
+  belongs_to :published_by, foreign_key: :user_id, class_name: 'User'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.